android/jni/com/mapswithme/maps/editor/Editor.cpp

```diff
@@ -247,7 +247,10 @@ Java_com_mapswithme_maps_editor_Editor_nativeSaveEditedFeature(JNIEnv *, jclass)
   case osm::Editor::SavedSuccessfully:
     return true;
   case osm::Editor::NoFreeSpaceError:
+  case osm::Editor::NoUnderlyingMapError:
+  case osm::Editor::SavingError:
     return false;
+  default: return false;
```

Comment:
In short: yes, I'm asking you to get rid of default case.

In detail: from my point of view there's a little difference (if at all) between undefined behavior and between broken program logic. When you forget to process some value of the enum, and blindly return false, the program logic is broken. This may lead to undefined behavior, as you can't reason about program anymore. On the the other hand, we work intensively on our project and, therefore, the we build project quite often. Compiler warnings can be easily spot by an eye, and, of course, should be fixed ASAP, especially such important warnings as unhandled case values.